radio/src/lua/api_general.cpp

```diff
@@ -438,15 +438,15 @@ When called without parameters, it will only return the status of the output buf
 
 @param value     value
 
-@retval boolean  data queued in output buffer or not.
+@retval boolean  data queued in output buffer or not. nil if incorrect telemetry protocol.
 
 @status current Introduced in 2.2.0
```

Comment:
You need to mention the update here (2.3.4)

Reply:
Done!
Updated  ``@status`` and moved the initial comment to a new line with ``@retval nil`` to make things a little clearer.